<launch>
  <rosparam command="load" file="$(find zumo_pi)/config/joy-teleop.yaml" />
  <node pkg="joy" name="joy" type="joy_node">
    <param name="dev" value="/dev/input/js0" />
    <!-- its a percentage value between 1 and 0-->
    <param name="deadzone" value="0.05" />
    <!-- HZ rate of resending even when the joy didn't change. double type:  0.0 == disable -->
    <param name="autorepeat_rate" value="1.0" />
    <!-- value in seconds to bundle axis events in a single message -->
    <param name="coalesce_interval" value="0.1" />
  </node>
  <node pkg="zumo_pi" name="amg8833_to_image" type="amg8833_to_image.py">
  </node>
  <node pkg="teleop_twist_joy" name="teleop_twist_joy" type="teleop_node">
    <param name="~scale_linear" value="0.75" />
    <param name="~scale_angular" value="0.75" />
    <!-- Scale to apply to joystick angular axis for regular-speed movement, in rad/s.-->
  </node>
  <node pkg="joy_teleop" name="joy_teleop" type="joy_teleop.py">
  </node>  
</launch>	

